modules/update-duration-example.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/understanding-openshift-update-duration.adoc
+
+:_content-type: CONCEPT
+[id="update-duration-example_{context}"]
+= Example update duration of cluster Operators
+
+image::update-duration.png[A diagram displaying the periods during which cluster Operators update themselves during an OCP update]
+
+The previous diagram shows an example of the time that cluster Operators might take to update to their new versions.
+The example is based on a three-node AWS OVN cluster, which has a healthy compute `MachineConfigPool` and no workloads that take long to drain, updating from 4.13.0 to 4.14.0.
+
+[NOTE]
+====
+The specific update duration of a cluster and its Operators can vary based on several cluster characteristics, such as the target version, the amount of nodes, and the types of workloads scheduled to the nodes.
```

Comment:
> Did you notice this doc doesn't just quote you a number?  Yeah, we're  about to explain a bunch of dynamic factors below :)

We also have "OpenShift Container Platform update duration varies based on the deployment topology" at the top of the doc.  But I agree that it's worth reminding folks a few times in this doc that the durations are variable based on multiple factors, so I'm just calling out your note as both somewhat redundant, and still useful anyway :).

Reply:
Lol fair, I wanted some caveat near the image that technically introduced some concrete-ish numbers. If you're cool with keeping it I'd like to keep it, but if other reviewers call out that it's overly redundant as well, I can remove it.

Reply:
I'm fine keeping it.  I agree that a local disclaimer near the image makes sense, even though we have similar disclaimers in other places.